app/locale/nl-NL.coffee

```diff
@@ -551,11 +551,11 @@ module.exports = nativeDescription: "Nederlands (Nederland)", englishDescription
 
   subscribe:
     comparison_blurb: "Verbeter je vaardigheden met een abonnement op CodeCombat!"
-    feature1: "meer dan $t(data.levelsCount) basislevels over $t(data.worldsCount) werelden" # {change}
-    feature2: "$t(data.heroesCount) sterke <strong>nieuwe helden</strong> met unieke vaardigheden!" # {change}
-    feature3: "$t(data.bonusLevelsCount) bonuslevels" # {change}
+    feature1: "meer dan _data.levelsCount_ basislevels over _data.worldsCount_ werelden" # {change}
+    feature2: "_data.heroesCount_ sterke <strong>nieuwe helden</strong> met unieke vaardigheden!" # {change}
+    feature3: "_data.bonusLevelsCount_ bonuslevels" # {change}
```

Comment:
Only one thing left: you should take the 'data.' part out and add an extra underscore so that there are two underscores before and after the word (like in the English version).

@nwinter, can't this conversion be made with a script?